haskell-indexer-pipeline-ghckythe-wrapper/main/GhcKytheWrapper.hs

```diff
@@ -0,0 +1,34 @@
+-- Copyright 2017 Google Inc.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+{- | Runs the indexer and emits serialized Kythe storage protos to stdout.
+The individual protos are prefixed by a varint-encoded binary length.
+
+Example: Indexing a module.
+
+    $ ghc_wrapper -c mycorpus -P gcc -- A.hs \
+          | /opt/kythe/tools/write_entries -graphstore /tmp/gs
+    $ /opt/kythe/tools/http_server \
+          -graphstore /tmp/gs \
+          -listen 0.0.0.0:8000 \
+          -public_resources /opt/kythe/web/ui
+
+Note: the arguments after the '--' are standard GHC arguments.
+-}
+module Main where
+
+import Language.Haskell.Indexer.Args
```

Comment:
optional: explicitly import (wrapperMain). Here the two modules are pretty close, but not a big effort and generally a nice thing to do (removes ambiguities for the reader).